lib/livebook_web/live/home_live/edit_sessions_component.ex

```diff
@@ -0,0 +1,73 @@
+defmodule LivebookWeb.HomeLive.EditSessionsComponent do
+  use LivebookWeb, :live_component
+
+  @impl true
+  def render(assigns) do
+    ~H"""
+    <div class="p-6 pb-4 flex flex-col space-y-8">
+      <h3 class="text-2xl font-semibold text-gray-800">
+        <%= title(@action) %>
+      </h3>
+      <.message action={@action} selected_sessions={@selected_sessions} sessions={@sessions}/>
+      <div class="mt-8 flex justify-end space-x-2">
+        <button class="button-base button-red" phx-click={@action} phx-target={@myself}>
+          <.remix_icon icon="close-circle-line" class="align-middle mr-1" />
+          <%= button_label(@action) %>
+        </button>
+        <%= live_patch "Cancel", to: @return_to, class: "button-base button-outlined-gray" %>
+      </div>
+    </div>
+    """
+  end
+
+  defp message(%{action: "close_all"} = assigns) do
+    ~H"""
+    <p class="text-gray-700">
+      Are you sure you want to close <%= pluralize(length(@selected_sessions), "session", "sessions") %>?
+      <%= if not_persisted(@selected_sessions) > 0 do %>
+      <br/>
+        <span class="font-medium">Important:</span>
+        <%= pluralize(
+          not_persisted(@selected_sessions),
+          "notebook is not persisted and its content will be lost.",
+          "notebooks are not persisted and their content will be lost."
```

Comment:
We are actually persisting all notebooks _somewhere_ now, so this is no longer true, but in this case it's probably ok to warn too much rather than not. Alternatively we can skip the "content will be lost" part, @josevalim wdyt?

Reply:
What about "will be lost" to "may be lost"?

Reply:
For example, if you are running on Fly and then you do a new deploy the autosave path may be gone?

Reply:
Prefect!